### DISCUSSION TITLE
Macos iio.h missing

### DISCUSSION THREAD

**Discussion by loewal** [AlexandreRouma/SDRPlusPlus#411] [Q&A]
Hello Alexandre,
I'm running Macos Monterey.
- installed libiio via homebrew
- installed: 
git clone https://github.com/analogdevicesinc/libiio.git
cd libiio
git reset --hard v0.21
mkdir build; cd build
cmake -Wno-dev ..

And yet it is not found.
/Users/Loewal/Downloads/SDRPlusPlus/source_modules/plutosdr_source/src/main.cpp:8:10: fatal error: 'iio.h' file not found
#include <iio.h>

What can I do pleaase?

Thank you for your great program!

**Comment by KyotoFox** [comment]
This is more of a workaround than a fix, but I ran into the same issue, and I don't need support for plutosdr, so I just added `-DOPT_BUILD_PLUTOSDR_SOURCE=OFF` to `cmake`, and the build finished without issues for me.
